### ISSUE TITLE
Mobile oddities

### ISSUE THREAD

**Issue by ebrent8** [MinnowBoard-org/website#418]
Viewing the production public website on iPhone 5s, possibly other mobile devices.

1.  There is a strange letter "**g. |**" on the top right of the screen in mobile mode.  See screenshot.

2.  The "**goto project**" popup appeared/disappeared despite not being logged in to GitHub on my phone.

![img_3371](https://cloud.githubusercontent.com/assets/18618930/24066068/cc910c02-0b2c-11e7-9241-9d0296b39b8f.PNG)
![img_3372](https://cloud.githubusercontent.com/assets/18618930/24066069/ccedb9ca-0b2c-11e7-9fb6-baa4bce2ea77.PNG)

**Comment by keerockl** [comment]
It looks fine on my device. I am using LG G3 Android Phone. browser is Chrome.

**Comment by jketreno** [comment]
Some data to help in prioritizing the implementation and research into this issue. On average, 13% of traffic to the site come in on mobile. During social network ad campaigns, that percentage climbs to ~33%. When mobile does come in, approximately 71% of the mobile traffic originates from Android devices, and ~27% from iOS.

As a minimal step, we should look to add es6-shim to see if that fixes any iOS issues.

The "goto project" is not related to being logged into GitHub; it is a dialog which is intended to be hidden if window.location.hostname resolves to minnowboard.org. The URL indicates it should be matching that, so I'm not sure what is causing the dialog to appear. It could be an ES5/ES6 compatiability issue? The relevant code is in index.html:

```javascript
      ready: function() {
        ...
        /* Only show version string on non-production domains */
        if (window.location.hostname == 'minnowboard.org') {
          return;
        }
```

**Comment by ebrent8** [comment]
Yeah.  I think this is a pretty low priority compared to other issues.  Not the biggest deal, I just wanted to note it somewhere.
